lib/setup/setupInstall.js

```diff
@@ -1444,8 +1444,8 @@ function Install(options) {
             adapter, adapter + '.admin'
         ]) {
             try {
-                const obj = await delObjectAsync(objId);
-                obj && console.log(`host.${hostname} object ${objId} deleted`);
+                await delObjectAsync(objId);
```

Comment:
@Apollon77 here, it is used with the assumption that `delObjectAsync(...)` resolves to an object, which it doesn't